Parser/lexer/state.h

```diff
@@ -59,6 +59,8 @@ typedef struct _tokenizer_mode {
     char* last_expr_buffer;
     int f_string_debug;
     int in_format_spec;
+
+    int tstring;
```

Comment:
I would recommend an enum (like type that has FSTRING/TSTRING) instead of having a boolean for tstring.